src/panels/SkyBrowserPanel/Tabs/SelectedImageCard.tsx

```diff
@@ -0,0 +1,97 @@
+import { Group, Image, Paper, Slider, Stack, Text } from '@mantine/core';
+import { useThrottledCallback } from '@mantine/hooks';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { MoveTargetIcon, TrashIcon } from '@/icons/icons';
+import { useAppSelector } from '@/redux/hooks';
+import { SkyBrowserImage } from '@/redux/skybrowser/skybrowserSlice';
+
+import { useActiveImage, useSelectedBrowserColorString } from '../hooks';
+import { ImageInfoPopover } from '../ImageInfoPopover';
+
+import { TabButton } from './TabButton';
+
+interface Props {
+  image: SkyBrowserImage;
+  selected: boolean;
+  opacity: number;
+}
+export function SelectedImageCard({ image, opacity }: Props) {
+  const luaApi = useOpenSpaceApi();
+  const [activeImage, setActiveImage] = useActiveImage();
+  const selectedBrowserId = useAppSelector((state) => state.skybrowser.selectedBrowserId);
+  const color = useSelectedBrowserColorString();
+
+  const throttle = useThrottledCallback(
+    (newValue) =>
+      luaApi?.skybrowser.setOpacityOfImageLayer(selectedBrowserId, image.url, newValue),
+    250
+  );
+  const isSelected = activeImage === image.url;
+
+  return (
+    <Paper
+      withBorder={true}
+      my={'md'}
+      p={'sm'}
+      style={{
+        borderColor: isSelected ? color : 'var(--mantine-color-gray-8)'
+      }}
+    >
+      <Group w={'100%'} justify={'space-between'}>
+        <Stack maw={'50%'}>
+          <Text fw={600} lineClamp={1}>
+            {image.name}
+          </Text>
+          <Group>
+            <TabButton
+              onClick={() => {
+                luaApi?.skybrowser.selectImage(image.url);
+                setActiveImage(image.url);
+              }}
+              text={'Look at image'}
+              variant={'filled'}
+            >
+              <MoveTargetIcon />
+            </TabButton>
+            <TabButton
+              onClick={() =>
+                luaApi?.skybrowser.removeSelectedImageInBrowser(
+                  selectedBrowserId,
+                  image.url
+                )
+              }
+              text={'Remove image'}
+              variant={'filled'}
```

Comment:
How about making this a red outlined button and moving it to the farthest right of the three buttons?

Among the actions (focus image, read about image, and delete) this seems like the one that comes least in the "hiearachy"